### PR TITLE
README Updates

### PR DESCRIPTION
**Pull request by prsimp**
Made some updates. Let me know what you think, particularly about the changes in https://github.com/dhstewart/Gilded-Rose-Ruby/commit/8d06b19e8ac05a09479ab196eaadc1c17aeaf94f.

/cc @dhstewart @steveforse @nathanlrivera

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +0,0 @@
-2.2.4
```
Comment (prsimp) [inline .ruby-version:1]: This seems to trip people up. If we run into compatibility issues we can re-address to figure out a better approach.

File: README.md [hunk 2]
```diff
@@ -10,25 +10,25 @@ Hi and welcome to team Gilded Rose. As you know, we are a small inn with a prime
...
 
-Pretty simple, right? Well this is where it gets interesting:
+#### Pretty simple, right? Well this is where it gets interesting:
 
 - Once the sell by date has passed, Quality degrades twice as fast
 
 - The Quality of an item is never negative
 
-- "Aged Brie" actually increases in Quality the older it gets
+- "Aged" items actually increases in Quality the older they get
```
Comment (nathanlrivera) [inline README.md:19]: This should read `"Aged" items actually increase in Quality`

**Review by nathanlrivera — APPROVED** [review]
lgtm, I noted a typo in the readme

**Review by dhstewart — APPROVED** [review]
LGTM, thanks for taking the time on this

File: .ruby-version (hunk 1 above)
Comment (steveforse) [reply .ruby-version:1]: Agreed. This has never come up in a helpful way that I can remember.

**Pull request merged**